table.go

```diff
@@ -291,8 +291,13 @@ func (t *Table[T]) DefaultDraw(canvas *Canvas, dirty Rect) {
 						canvas.Rotate(90)
 						canvas.Translate(-offset, -offset)
 					}
-					canvas.DrawPath(CircledChevronRightSVG.PathForSize(dSize),
-						fg.Paint(canvas, cellRect, paintstyle.Fill))
+					chevron := DrawableSVG{
+						SVG: CircledChevronRightSVG,
+					}
+					chevronPaint := NewPaint()
+					chevronPaint.SetColor(Black)
```

Comment:
This should use the color of `fg`. But `fg` is an `Ink` and I can't figure out to do it.

Reply:
I've reinstated the old code now that I've added missing `SVG` methods.